__tests__/rpcProvider.test.ts

```diff
@@ -1,17 +1,61 @@
-import { RpcProvider } from '../src';
-import { describeIfRpc, getTestProvider } from './fixtures';
+import { isBN } from 'bn.js';
+
+import { Account, RpcProvider, ec } from '../src';
+import {
+  compiledErc20,
+  compiledOpenZeppelinAccount,
+  describeIfRpc,
+  getTestAccount,
+  getTestProvider,
+} from './fixtures';
 
 describeIfRpc('RPCProvider', () => {
-  let provider: RpcProvider;
+  const provider = getTestProvider() as RpcProvider;
+  const account = getTestAccount(provider);
+  let erc20Address: string;
+  let accountPublicKey: string;
 
   beforeAll(async () => {
-    provider = getTestProvider() as RpcProvider;
+    expect(account).toBeInstanceOf(Account);
+
+    const erc20Response = await provider.deployContract({
+      contract: compiledErc20,
+    });
+
+    erc20Address = erc20Response.contract_address;
```

Comment:
This whole erc20 part is redundant (because of https://github.com/0xs34n/starknet.js/pull/309/files#r959545687)
So we can remove them and simplify this whole `beforeAll` part